release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3202,3 +3202,24 @@ Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which s
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-42"]
+=== RHBA-2021:3008 - {product-title} 4.6.42 bug fix and security update
+
+Issued: 2021-08-11
+
+{product-title} release 4.6.42 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3008[RHBA-2021:3008] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:3009[RHBA-2021:3009] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6249631[{product-title} 4.6.42 container image list]
+
+[id="ocp-4-6-42-bug-fixes"]
+==== Bug fixes
+
+* Previously, host-network pods had IPv4 IPs making them unreachable from IPv6-only pods. This update fixes the `node-IP-picking code` so nodes will have IPv6 IPs, so now the host-network pods will be reachable.(https://bugzilla.redhat.com/show_bug.cgi?id=1942506[*BZ1942506*])
```

Comment:
IBM doesn't like future tense (like "will have"). Reword this to something like "This update fixes the `node-IP-picking-cod` so that nodes have IPv6 IPs. As a result, the host-network pods are now reachable." 

See https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense